package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -607,14 +608,38 @@ def _read_next_timestep(self):
         coord_cols = convention_to_col_ix[self.lammps_coordinate_convention]
 
         ids = "id" in attr_to_col_ix
+
+        # Create the data arrays for additional attributes which will be saved 
```

Comment:
As stated above, I would add a kwarg to `__init__` meaning you have to specify which additional columns you want saved in `ts.data`. Meaning you can prefill `ts.data` and remove a bit of this loop.